textproc/libxslt/Portfile

```diff
@@ -28,15 +28,14 @@ use_xz          yes
 
 depends_build   port:pkgconfig
 
-patchfiles      dynamic_lookup-11.patch \
```

Comment:
Even though there does not appear to be a build failure, I would like to keep this patch until they release a version of libxslt whose configure script was generated with a new enough libtool version so that it already contains this fix. The patch will need to be updated for this new libxslt version.

Reply:
FWIW if I autoreconf I get
```
 9702     darwin1.*)
 9703       _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
 9704     darwin*)
 9705       case $MACOSX_DEPLOYMENT_TARGET,$host in
 9706         10.[012],*|,*powerpc*-darwin[5-8]*)
 9707           _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
 9708         *)
 9709           _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
 9710       esac
 ```
 
 Would you rather autoreconf here than maintain the patch?

Reply:
I would rather maintain the patch. Patching is quicker than running autoreconf.